src/orange/detail/UnitInserter.cc

```diff
@@ -187,6 +218,17 @@ SimpleUnitId UnitInserter::operator()(UnitInput const& inp)
         make_builder(&orange_data_->volume_records)
             .insert_back(vol_records.begin(), vol_records.end()));
 
+    // Create BIH tree
+    if (std::all_of(
+            bboxes.begin(), bboxes.end(), [](FastBBox const& b) { return b; }))
+    {
+        unit.bih_params = bih_builder_(std::move(bboxes));
+    }
+    else
+    {
+        // TODO: Handle case where geometry does not have valid bounding boxes
+    }
+
```

Comment:
If I recall, we agreed that it's OK to require all bboxes (since there's only a couple of tests where it would be convenient not to):
```suggestion
    // Create BIH tree
    CELER_VALIDATE(std::all_of(bboxes.begin(),
                               bboxes.end(),
                               [](FastBBox const& b) { return b; }),
                   << "not all bounding boxes have been assigned");
    unit.bih_params = bih_builder_(std::move(bboxes));

```

Reply:
This breaks 33 tests. I can fix it, but it's going to touch a lot of files in an already large PR...

Reply:
Oh jeez, I think I misremembered then. Sorry!